consult.el

```diff
@@ -4405,6 +4415,25 @@ FIND-FILE is the file open function, defaulting to `find-file'."
       (substring cand (1+ (length (get-text-property 0 'consult--grep-file cand))))
     (get-text-property 0 'consult--grep-file cand)))
 
+(defun consult--grep-generate-exclude-args-common ()
+  "Produce grep exclude arguments based on `grep-find-ignored-files'."
+  (unless (boundp 'grep-find-ignored-files) (require 'grep))
+  (append
+   (mapcar (lambda (s) (concat "--exclude=" s))
+	   (bound-and-true-p grep-find-ignored-files))
+   (mapcar (lambda (s) (concat "--exclude-dir=" s))
+	   (bound-and-true-p grep-find-ignored-directories))))
+
+(defun consult--grep-generate-exclude-args-project ()
+  "Produce grep exclude arguments based on `project-ignores'."
```

Comment:
I don't want to provide that function here. Users can add this code to their user configuration.

Reply:
removed. Shall I add it to the wiki ?